src/sentry/spans/span.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import hashlib
+from dataclasses import dataclass
+from typing import Mapping, Optional, Sequence, TypedDict
+
+from sentry import nodestore
+
+
+class SpanData(TypedDict):
+    id: str
+    contexts: Mapping[str, object]
+    project_id: int
+    tags: Mapping[str, str]
+
+
+@dataclass(frozen=True)
+class Span:
+    contexts: Mapping[str, object]
```

Comment:
Let's rename to attributes.

The only open question is where will things like user/os/device information live? We have a separate schema for these than traditional otel attributes (https://develop.sentry.dev/sdk/event-payloads/contexts/)

I like renaming everything to be attributes, but we probably need some layer in Relay that maps all the old context -> span attributes accordingly, since otel span attributes are usually flat, and sentry contexts are nested.